examples/gno.land/p/demo/pausable/helper.gno

```diff
@@ -0,0 +1,9 @@
+package pausable
+
+import (
+	"std"
+)
+
+func prevRealmIsOwner(addr std.Address) bool {
+	return std.PrevRealm().Addr() == addr
```

Comment:
this is not checking ownership.

Reply:
https://github.com/gnolang/gno/pull/2198/commits/8bd3ac0bd411c9f2adee3bbc466e57ed6f33ac5b